libs/partners/ai21/langchain_ai21/llms.py

```diff
@@ -0,0 +1,141 @@
+import asyncio
+from functools import partial
+from typing import (
+    Any,
+    List,
+    Optional,
+)
+
+from ai21.models import CompletionsResponse, Penalty
+from langchain_core.callbacks import (
+    AsyncCallbackManagerForLLMRun,
+    CallbackManagerForLLMRun,
+)
+from langchain_core.language_models import BaseLLM
+from langchain_core.outputs import Generation, LLMResult
+
+from langchain_ai21.ai21_base import AI21Base
+
+
+class AI21(BaseLLM, AI21Base):
+    """AI21LLM large language models.
+
+    Example:
+        .. code-block:: python
+
+            from langchain_ai21 import AI21LLM
+
+            model = AI21LLM()
+    """
+
+    model: str = "j2-ultra"
```

Comment:
Depending how long you think this model will be SOTA, it's likely better to not have a default value here, and always document it with a model string. Otherwise changing default here will be a breaking change in the future.

Reply:
Fixed. Made it mandatory and added a doc string with a URL which lists the available models